src/main/NWN/API/Object/NwSound.cs

```diff
@@ -26,5 +26,10 @@ public sbyte Volume
     /// Stops this sound object from playing.
     /// </summary>
     public void Stop() => NWScript.SoundObjectStop(this);
+
+    /// <summary>
+    /// Set the location of this sound object.
+    /// </summary>
+    public new void Location(Location destination) => NWScript.SoundObjectSetPosition(this, destination.Position);
```

Comment:
This should be a property, and override Position instead as this does not use the location's area or rotation.